core/src/toga/style/__init__.py

```diff
@@ -1,2 +1,2 @@
 from toga.style.applicator import TogaApplicator  # noqa: F401
-from toga.style.pack import Pack  # noqa: F401
+from toga.style.pack import Pack as Pack  # noqa: F401
```

Comment:
Oh...i should have just left this out until later. This is just something stricter `mypy` gets upset about because `toga.style` is not the owner of this object. So, when other projects import `Pack` from `toga.style`, it is considered an implicit re-export. I imagine the concern is that since `toga.style` doesn't own `Pack`, it could be renamed and the symbol would disappear from this namespace potentially without really understanding the consequence (although, its a bit obvious in this example). For instance, someone could have even aliased the new name of the object to `Pack` back over in `toga.style.pack` but maybe they won't here. So, doing an explicit re-export via `as Pack` helps explicitly avoid that situation.

This is one of mypy's stricter modes that is enabled via `--no-implicit-reexport`.

Reply:
Ah - Ok. That makes sense; and oddly, I'm actually *more* OK with it given that's the reason, as it requires us to beexplicit about thing we're re-exporting.

Reply:
The `--no-implicit-reexport` docs say this can also be done using `__all__`, which seems like a less confusing and more explicit solution.